src/main/java/org/mongojack/DBUpdate.java

```diff
@@ -658,6 +660,19 @@ public DBObject serialiseAndGet(ObjectMapper objectMapper,
                     javaType);
         }
 
+        /**
+         * Serialise the values of the query and get them
+         * 
+         * @param objectMapper
+         *            The object mapper to use to serialise values
+         * @return The object
+         */
+        public Document serialiseAndGetAsDocument(ObjectMapper objectMapper,
```

Comment:
typo - serialize

Reply:
Darn. Will fix in the morning.

Reply:
Should be noted this is also mispelled in the existing (DBObject) method, but I think seriazeAndGet is meant for internal use only.